cmake/install.cmake

```diff
@@ -122,19 +122,19 @@ configure_file(protobuf-options.cmake
 
 if (protobuf_BUILD_PROTOC_BINARIES)
   export(TARGETS libprotobuf-lite libprotobuf libprotoc protoc
-    NAMESPACE protobuf::
+    NAMESPACE ${nameSpace}
     FILE ${CMAKE_INSTALL_CMAKEDIR}/protobuf-targets.cmake
   )
 else (protobuf_BUILD_PROTOC_BINARIES)
   export(TARGETS libprotobuf-lite libprotobuf
-    NAMESPACE protobuf::
+    NAMESPACE ${nameSpace}
     FILE ${CMAKE_INSTALL_CMAKEDIR}/protobuf-targets.cmake
```

Comment:
previous [patch](https://github.com/protocolbuffers/protobuf/compare/v3.0.0-beta-1...smanders:xp3.0.0-beta-1#diff-8e43682de5cdf140cf423aec73d89ee4528eded5a9eee9083a9d9ab4f755b7b8) had `DESTINATION "lib/cmake/protobuf${ver}"` -- I will be checking if `CMAKE_INSTALL_CMAKEDIR` keeps `${ver}` in the mix and if we still need that...